server/app.py

```diff
@@ -97,18 +97,18 @@ def main():
         config = configparser.ConfigParser()
         config.read(tornado.options.options.cfg)
 
-    # prepare database
-    logger.info("Preparing database...")
-    database = db.DataBase()
-    database.init()
-
     options_dict = tornado.options.options
+    logging.info(f'The app is running and listening on port {options_dict.server_port}')
```

Comment:
The app starts later near `app_instance.start()` line. Could you please move this logging here?